projects/plugins/jetpack/to-test.md

```diff
@@ -1,40 +1,31 @@
-- Select "This is a personal site".
-- Then select "Continue", it should skip straight to the Downtime Monitoring recommendation.
-- Navigate back to `/wp-admin/admin.php?page=jetpack#/recommendations/site-type`
-- This time, select the "I build or manage this site for a client" option.
-- Select "Continue" and this time you should get the new agencies recommendation.
-- On the agency screen, test that the "Learn More" and "Get Jetpack for Agencies" links work.
-- Click "Not now" on the agency screen recommendation and you should be directed to the next recommendation. Continue selecting "Not now" until you reach the summary page.
-- Click the "Sign Up" external link next to the Jetpack for Agencies recommendation and make sure it opens (no need to fill out): https://cloud.jetpack.com/agency/signup
-- Navigate back to `/wp-admin/admin.php?page=jetpack#/recommendations/site-type` and select both "I build or manage this site for a client" and "This is an e-commerce site" and click on Continue.
-- You should see the Agencies recommendation first, select "Not now".
-- Next you should see the WooCommerce recommendation.
-- That is all for testing these updated recommendations.
-
-**And to test the mobile app links:**
-
-- Navigate to `/wp-admin/admin.php?page=jetpack#/recommendations/summary`
-- On a desktop or laptop computer, a QR code for the Jetpack mobile app will be displayed next to the `jetpack.com/mobile` URL.
-- When visiting the same recommendations page on a mobile device, the respective app store button should be shown for well known devices and mobile operating systems.
-- If you don't have a mobile device to test with, you might try using Chrome to simulate the user agent:
-  - Open the Chrome Dev Tools
-  - Click the vertical menu option > More tools > Network conditions
-  - In the Network conditions tab, uncheck the "Use browser default" for the user agent, and select an iOS or Android one for testing.
-  - When refreshing the recommendations page you should see the expected mobile app store button.
+### Jetpack Forms block
+
+Forms block got several updates and bug fixes in this version. To test, try the following:
+
+- Make sure the testing site has Jetpack connected to your account.
+- Create a post and add a Form block to it. 
+- Hover your pointer between two Form child blocks (eg. Name and Email in Contact form). The Block Inserter (plus sign with Add block title) should appear, allowing you to easily insert new fields between the blocks.
+- Check the Dimensions section in the sidebar and make sure it already has padding setting (16 px). Try editing padding.
+- Try editing Field widths for different fields and make sure that what you see in the editor matches the frontend.
+- Add a Contact form and select the Form block container. Make sure the edit button doesn't show up (pen icon). All form settings such as email address and subject line should be available in the sidebar.
```

Comment:
> Add a Contact form

Is this a new form block being added now, not clear among the steps to me?

Reply:
No, this is the old one but is getting a lot of attention in this version.

Reply:
Sorry, I meant second form block being added in the testing steps since it starts off with "Create a post and add a Form block to it." then later on the "Add a Contact form".

If it's a second form block being added, might help to break things up a bit.